cluster/plan.go

```diff
@@ -879,20 +878,5 @@ func getUniqStringList(l []string) []string {
 }
 
 func (c *Cluster) getRKEToolsEntryPoint() string {
-	v := strings.Split(c.SystemImages.KubernetesServicesSidecar, ":")
```

Comment:
This is required to support older 1.9.x and 1.10.x clusters from rancher 2.0 line.

Reply:
oh. they can't work with newer rke-tools version?

Reply:
TL;DR: No. 

We have two rke-tools "milestones":
- At one point, we had to change the entry point to fix a bug. For that we created `getRKEToolsEntryPoint()` which will check if the clusters rke-tools version is > or <= `0.1.13` and return an entry point based on that version. Most of the 2.0.x line clusters are `0.1.13` or less. Today I saw several clusters with `0.1.8`! This workaround was meant to be deprecated once we EOL those versions, but some users are still using them. So, it's safer to keep it around for a bit longer. 

- When we added rancher etcd backup, we maintained backward comparability with rke backup UI, but several internal changes had to be done to support s3 backup. This changed how `rke-tools` is used. So,  we created `util.IsRancherBackupSupported()` and the related logic to handle this. this way clusters with older `rke-tools` version that doesn't support s3 backup doesn't have to be updated to the latest k8s/rke-tools version, but still can utilize rancher s3 backup. Using this logic, rancher will pull the the default `rke-tools` and use it for backup operations without upgrading the cluster `system_images`. If the image is support the new backup api, it will be simply used instead of pulling a different image.

Reply:
I was trying to avoid back porting and introducing K8s versions just because rke-tools changed, since they are not tied to kubernetes version. Based on these milestones, looks like we'll have to continue doing it for upgraded setups. 

Would it be helpful if we handled this logic this way? 

1) new clusters 
- rancher won't pass rke-tools version, so it'll only be dependent on rke version 

2) upgraded clusters 
- rancher uses the new driver metadata information to read rke-tools for k8s version and passes this in system images as before. I think this way we shouldn't need `util.IsRancherBackupSupported()` logic